clock.json

```diff
@@ -249,6 +293,54 @@
             "expected": true
          },
          {
+            "description": "clocks with hour overflow by several days",
+            "clock1": {
+               "hour": 3,
+               "minute": 11
+            },
+            "clock2": {
+               "hour": 99,
+               "minute": 11
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hour",
+            "clock1": {
+               "hour": 22,
+               "minute": 40
+            },
+            "clock2": {
+               "hour": -2,
+               "minute": 40
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hour that wraps",
```

Comment:
Ah, no, I think I meant to do something to wrap exactly one day, so -31 instead of -7. Let me push up a fix.